src/main/java/seedu/ui/Ui.java

```diff
@@ -55,20 +55,19 @@ public class Ui {
             HORIZONTAL_LINE +
             "\n7. List all expenditures and their total amount in the record BY CATEGORY\n"
             + "Format: viewtype CATEGORY\n"
-            + "There are 7 EXPENDITURE_TYPE: 'Academic', 'Accommodation', 'Entertainment'," +
-            "'Food', 'Transport', 'Tuition', 'Other'.\n" +
-            "  This includes 'Borrow' and 'Lend'.\n"
+            + "There are 9 CATEGORIES: 'academic', 'accommodation', 'entertainment'," +
+            "'food', 'transport', 'tuition', 'other', 'borrow', 'lend'\n" 
             + "Example: viewtype food\n" +
             HORIZONTAL_LINE +
             "\n8. Set a fixed budget for money management\n"
             + "Format: set AMOUNT\n"
             + "Example: set 150\n" +
             HORIZONTAL_LINE +
-            "\n9. Check expenditure with respect to set allocated budget\n"
-            + "Format: check\n"
-            + "This command helps you manage your budget to check if you are overspending" +
-            " and how much leeway you have!\n"
-            + "Example: viewdate 2023-03-29\n" +
+            "\n9. Check expenditure with respect to a allocated budget by date(yyyy-mm-dd) or by year(yyyy)\n"
+            + "Format: check d/DATE\n"
+            + "Format: check y/YEAR\n"
+            + "Example of checking budget by date: viewdate d/2023-03-29\n" 
+            + "Example of checking budget by year: viewdate y/2023\n" +
```

Comment:
`check` is missing the `check by expenditure type`. Should include those as well.